README.md

```diff
@@ -2386,6 +2389,12 @@ Optics Letters, 2021.
 *[Lens-free motion analysis via neuromorphic laser speckle imaging](https://doi.org/10.1364/OE.444948)*,  
 Optics Express, 2022.
 
+<a name="coherence_scanning_interferometry"></a>
+## Coherence Scanning Interferometry
```

Comment:
Since this subsection has only 1 entry and the previous one has also only 4, would you consider merging them, potentially together with other entries into something like "Optics Applications" ? @kqwang

Reply:
I agree. I merged all new paper I added before into one part as "Optical Applications".